src/toolkits/drawing_classifier/drawing_classifier.hpp

```diff
@@ -68,7 +68,7 @@ class EXPORT drawing_classifier : public ml_model_base {
                              variant_type validation_data,
                              std::map<std::string, flexible_type> opts);
 
-  virtual void iterate_training();
+  virtual void iterate_training(bool show_loss);
```

Comment:
We'll have to resolve how to deal with this argument when we expose this method for checkpointing, but this is okay for now.

Reply:
Okay!